storage/innobase/include/trx0types.h

```diff
@@ -146,6 +146,7 @@ typedef ib_id_t	undo_no_t;
 /** Transaction savepoint */
 struct trx_savept_t{
 	undo_no_t	least_undo_no;	/*!< least undo number to undo */
+	bool		vtq_notify_on_commit; /*!< Notify VTQ for System Versioned update */
```

Comment:
What does this mean? Add an elaborate comment. And add it with `/**`, preceding the field definition, indented by one TAB only. I do not like the write-in-the-margins style.

When is this flag set?

There are 3 kinds of rollbacks in InnoDB:
1. rollback of the latest inserted or updated row (say, on UNIQUE violation)
2. rollback to the start of the current statement, or ROLLBACK TO SAVEPOINT (both invoked by the SQL layer)
3. full ROLLBACK of the entire transaction

The comment should say in which of these scenarios this field applies and how. I would presume that it should only matter at full ROLLBACK.

Do we need this if you implement the `ha_commit_trans()` change that I suggested?